3]. Competitive Programming/06]. CodeChef/2]. Compete/4]. Starters/July/Cricket Ranking.cpp

```diff
@@ -0,0 +1,70 @@
+
+
+
+/* 
+    Solution by Rahul Surana
+    
+    ***********************************************************
+
+
+In a season, each player has three statistics: runs, wickets, and catches. 
+Given the season stats of two players A and B, denoted by R, W, and C respectively, the person who is better than the other in the most statistics is regarded as the better overall player. 
+Tell who is better amongst A and B. It is known that in each statistic, the players have different values.
+
+Input Format:
+
+The first line contains an integer T, the number of test cases. Then the test cases follow.
+Each test case contains two lines of input.
+The first line contains three integers R1, W1, C1, the stats for player A.
+The second line contains three integers R2, W2, C2, the stats for player B.
+
+Output Format:
+
+For each test case, output in a single line "A" (without quotes) if player A is better than player B and "B" (without quotes) otherwise.
+
+
+
+    ***********************************************************
+*/
+
+
+
```

Comment:
Check this code
And Also make a folder `Starters - 6` in place of month
like- `4]. Starters/01]. Starters - 6/<CODE_NAME>.cpp